internal/pkg/manifest/workload_applyenv_test.go

```diff
@@ -0,0 +1,1846 @@
+package manifest
+
+import (
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ApplyEnv_Image(t *testing.T) {
```

Comment:
nit: can we rename the tests to follow the format
```suggestion
func TestApplyEnv_Image(t *testing.T) {
```